config/webpack.config.js

```diff
@@ -425,9 +425,9 @@ module.exports = async (env = {}) => {
 
   if (!buildOptions.watch) {
     baseConfig.plugins.push(
-      new ManifestPlugin({
+      new WebpackManifestPlugin({
         fileName: 'file-manifest.json',
-        filter: ({ path: filePath }) => !filePath.includes('/generated/..'),
+        filter: ({ isChunk }) => isChunk,
```

Comment:
Our old filtering wasn't excluding fonts and svgs after upgrading the plugin for some reason. The mappings for those are still correct, but since we don't need them in  the manifest, I updated this to exclude them.